mikeio/dfsu/_spectral.py

```diff
@@ -177,6 +182,19 @@ def _read_geometry(
         dfs.Close()
         return geometry
 
+    @staticmethod
+    def _get_direction_unit(filename: str) -> int:
+        """Determine if the directional axis is in degrees or radians"""
+        source = DfsFileFactory.DfsGenericOpen(filename)
+        try:
+            for static_item in iter(source.ReadStaticItemNext, None):
+                if static_item.Name == "Direction":
```

Comment:
I think the names of the static items are just strings, but yeah I guess it has a EUM type as well, but since this is very specific for spectral dfsu files which are only produced by MIKE 21 SW, it should be safe enough to check for the name. Okay to keep it like that?

Reply:
Yea sounds reasonable to me 🚀 ... I guess also those names are quite stable and unlikely to change